app/views/agencies/_section3.html.erb

```diff
@@ -48,6 +50,8 @@
             </span>
           </a>
         </address>
+        <p><%= t'show_page.county' %> <small><%= agency.county ? agency.county : 'No info' %></small></p>
```

Comment:
Can you localize `'No info'` if this was in spanish the user would not understand.